### PR TITLE
Improve description of innovation fields in NKF4/XKF4 

### PR DESCRIPTION
**Pull request by peterbarker**
Courtesy Paul Riseborough here: https://github.com/ArduPilot/ardupilot_wiki/issues/3641#issuecomment-913577132

### CODE REVIEW

File: libraries/AP_NavEKF2/LogStructure.h [hunk 1]
```diff
@@ -166,7 +166,7 @@ struct PACKED log_NKF3 {
 
 
 // @LoggerMessage: NKF4
-// @Description: EKF2 variances
+// @Description: EKF2 variances  SV, SP, SH and SM are probably best described as 'Squared Innovation Test Ratios' where values <1 tells us the measurement was accepted and >1 tells us it was rejected. They represent the square of the (innovation / maximum allowed innovation) where the innovation is the difference between predicted andmeasured value and the maximum allowed innovation is determined from the uncertainty of the measurement, uncertainty of the prediction and scaled using the number of standard deviations set by the innovation gate parameter for that measurement, eg EK2_MAG_I_GATE, EK2_HGT_I_GATE, etc
```
Comment (joshanne) [inline libraries/AP_NavEKF2/LogStructure.h:169]: Space between "and measured". :)

```suggestion
// @Description: EKF2 variances  SV, SP, SH and SM are probably best described as 'Squared Innovation Test Ratios' where values <1 tells us the measurement was accepted and >1 tells us it was rejected. They represent the square of the (innovation / maximum allowed innovation) where the innovation is the difference between predicted and measured value and the maximum allowed innovation is determined from the uncertainty of the measurement, uncertainty of the prediction and scaled using the number of standard deviations set by the innovation gate parameter for that measurement, eg EK2_MAG_I_GATE, EK2_HGT_I_GATE, etc
```

File: libraries/AP_NavEKF3/LogStructure.h [hunk 2]
```diff
@@ -173,7 +173,7 @@ struct PACKED log_XKF3 {
 
 
 // @LoggerMessage: XKF4
-// @Description: EKF3 variances
+// @Description: EKF3 variances.  SV, SP, SH and SM are probably best described as 'Squared Innovation Test Ratios' where values <1 tells us the measurement was accepted and >1 tells us it was rejected. They represent the square of the (innovation / maximum allowed innovation) where the innovation is the difference between predicted andmeasured value and the maximum allowed innovation is determined from the uncertainty of the measurement, uncertainty of the prediction and scaled using the number of standard deviations set by the innovation gate parameter for that measurement, eg EK3_MAG_I_GATE, EK3_HGT_I_GATE, etc
```
Comment (joshanne) [inline libraries/AP_NavEKF3/LogStructure.h:176]: Ditto.

```suggestion
// @Description: EKF3 variances.  SV, SP, SH and SM are probably best described as 'Squared Innovation Test Ratios' where values <1 tells us the measurement was accepted and >1 tells us it was rejected. They represent the square of the (innovation / maximum allowed innovation) where the innovation is the difference between predicted and measured value and the maximum allowed innovation is determined from the uncertainty of the measurement, uncertainty of the prediction and scaled using the number of standard deviations set by the innovation gate parameter for that measurement, eg EK3_MAG_I_GATE, EK3_HGT_I_GATE, etc
```

**Review by joshanne — APPROVED** [review]

File: libraries/AP_NavEKF2/LogStructure.h (hunk 1 above)
Comment (peterbarker) [reply libraries/AP_NavEKF2/LogStructure.h]: Fixed, thanks!

File: libraries/AP_NavEKF3/LogStructure.h (hunk 2 above)
Comment (peterbarker) [reply libraries/AP_NavEKF3/LogStructure.h]: Ditto, thanks!

**Review by tridge — APPROVED** [review]

**Pull request merged**